test/cmd/test-scenario/generators.go

```diff
@@ -20,9 +20,19 @@ const (
 	envServiceCall = "SERVICE_CALL"
 )
 
+// Entry is a simple value object
```

Comment:
That's not helpful :) But I guess that's the problem of the godoc on its own hehe. Maybe we could say what is this entry about and for?

Reply:
https://github.com/Maistra/istio-workspace/pull/375/files#r393721620 ?